onnxruntime/core/providers/cpu/math/element_wise_ops.cc

```diff
@@ -240,9 +240,11 @@ REG_ELEMENTWISE_TYPED_KERNEL(Neg, 13, int64_t, Neg);
 
 REG_ELEMENTWISE_VERSIONED_TYPED_KERNEL(Floor, 6, 12, float, Floor);
```

Comment:
According to https://github.com/onnx/onnx/blob/main/docs/Changelog.md#Floor-6, the double is supported since opset verison 6 (opset 13 Floor added bfloat16). I think the following line can be added:
``
REG_ELEMENTWISE_VERSIONED_TYPED_KERNEL(Floor, 6, 12, double, Floor);
``

Reply:
Okay, I added the older-version kernels too!